.github/workflows/snyk-scan.yml

```diff
@@ -17,11 +21,12 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Run Snyk to check for vulnerabilities
-        uses: snyk/actions/golang@master
+        uses: snyk/actions/golang@cdb760004ba9ea4d525f2e043745dfe85bb9077e
```

Comment:
Would be great if there cloud be the comment with version tag of that commit as it is done here for example:
`uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2`